lib/fastlane/plugin/wpmreleasetoolkit/actions/configure/configure_apply_action.rb

```diff
@@ -10,6 +10,15 @@ module Fastlane
   module Actions
     class ConfigureApplyAction < Action
       def self.run(params = {})
+
+        ### Make sure secrets repo is at the proper hash as specified in .configure
+        repo_hash = Fastlane::Helper::ConfigureHelper.repo_commit_hash
+        file_hash = Fastlane::Helper::ConfigureHelper.configure_file_commit_hash
+
+        unless repo_hash == file_hash
+          sh("cd #{repository_path} && git fetch && git checkout #{file_hash}")
```

Comment:
What happens if this command fails? Does the script exit or carry on with the assumption that the repo is in the correct state?

Reply:
If `sh` fails, it logs an error to the console and exits.  The Fastlane docs say that `sh` takes an error callback function, so we could handle that case if we wanted to.

Reply:
If it exists, I'm happy with that. Thanks for confirming 👍